protocol-tests/src/main/java/fixtures/lro/implementation/LroRetrysImpl.java

```diff
@@ -325,7 +325,10 @@
         return PollerFlux.create(
                 Duration.ofSeconds(1),
                 () -> this.put201CreatingSucceeded200WithResponseAsync(requestOptions),
-                new DefaultPollingStrategy<>(this.client.getHttpPipeline()),
+                new DefaultPollingStrategy<>(
+                        this.client.getHttpPipeline(),
+                        null,
+                        requestOptions != null ? requestOptions.getContext() : Context.NONE),
```

Comment:
Should it be `requestOptions != null && requestOptions.getContext() != null ? requestOptions.getContext() : Context.NONE)`

I am not sure if `HttpClient.send(request, null)` works OK for all implementation of `HttpClient`.